source/SquadLeader/src/com/esri/squadleader/controller/AdvancedSymbolController.java

```diff
@@ -169,13 +203,27 @@ protected boolean processMessage(Geomessage geomessage) {
             message.setID(geomessage.getId());
         }
         
+        return _processMessage(message);
+    }
+    
+    private boolean _processMessage(Message message) {        
+        /**
+         * Workaround: ArcGIS Runtime 10.2.4 requires a chem light message to have
```

Comment:
Interesting - I did not know that - one could argue that is a bug - esp. since it is not even mentioned in the json: https://github.com/Esri/military-features-data/blob/master/data/mil2525c/messagetypes/Chemlight.json